Classes/Settings/SettingsVersionInfoSectionController.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  SettingsVersionInfoSectionController.swift
+//  Freetime
+//
+//  Created by Sherlock, James on 15/07/2017.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import UIKit
+import IGListKit
+
+final class SettingsVersionInfoSectionController: ListSectionController {
+    
+    override init() {
+        super.init()
+        inset = UIEdgeInsets(top: 0, left: 0, bottom: Styles.Sizes.tableSectionSpacing, right: 0)
```

Comment:
Bottom is unnecessary here, the lack of top is to make it closer to the log out button (hence the changes in the first place)

With the original spacing it was too far away which looked weird